setup.py

```diff
@@ -196,7 +196,7 @@ def read(*names, **kwargs):
         'ftfy>=  5.0.0; python_version > "3"',
 
         # scancode
-        'click >= 6.0.0, < 7.0.0',
+        'click >= 6.0.0, < 8.0.0',
```

Comment:
I would like go to a simpler `click >= 7.0.0` as they have a great track record of backward compatibility so far